UIMentoringSession/MenteesChallenges/Pets Main Card UI Chellenge/YefimYakunin/PetFilterCollectionCell/YeYa_PetFilterCollectionViewCell.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+final class YeYa_PetFilterCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet private weak var petIconImage: UIImageView!
+    @IBOutlet private weak var kindOfPetLabel: UILabel!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setupLayout()
+    }
+    
+    private func setupLayout() {
```

Comment:
order of methods

Reply:
final class YeYa_PetFilterCollectionViewCell: UICollectionViewCell {
    @IBOutlet private weak var petIconImage: UIImageView!
    @IBOutlet private weak var kindOfPetLabel: UILabel!
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        setupLayout()
    }

Corrected the order of methods